onadata/apps/api/viewsets/floip_viewset.py

```diff
@@ -77,6 +78,15 @@ def get_success_headers(self, data):
 
         return headers
 
+    def floip_responses_filter(self, request, queryset):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=4907810)